### ISSUE TITLE
File not found error on all custom list views

### ISSUE THREAD

**Issue by SuleymanA** [OfficeDev/PnP-Provisioning-Schema#53]
I just applied a provisioning XML to an existing site. XML contains site fields, content types and list instances. It all seem to be created fine except that the custom views in my list instances are not working. I receive File Not Found error on all views except for the default all items view. Problem disappears when I close the list view webpart via the webpart maintenance page. I do see the views in my list settings and can open and change the view configuration. 

I have tried closing the webpart and adding a fresh list view webpart which also resulted in the same error. I am able to add a new list view webpart to the view page without error after deleting the webpart that is causing the problem.

Do you have an idea of what can cause this and how it can be fixed?

**Comment by PaoloPia** [comment]
Hi,
Thanks for your feedback. Actually we do support custom views and "in general" they work. Can you share by email or on Yammer (http://aka.ms/OfficeDevPnPYammer) the template instance that you are using? Thanks. In the mean time I will keep this issue open.

**Comment by SuleymanA** [comment]
Hi Paola,

I performed provisioning hoping that it was just an incident. It failed a couple of times during list instance creating due to calculated columns. Was finally able to run the provisioning without any errors only to find that all lookup columns where not created correctly. They were all pointing to the template web (lookupwebid property). Realized this when I tried to use the lookup fields in my views. Strange thing was that it resulted in the same error as I described above. The lookup fields are presented correctly in the forms but when trying to use in a list view, it resulted in an error. This was probably also the case when I reported this issue. But for some reason no custom views were created this time at all... Probably because I re-run the provisioning without deleting all lists. So this issue can be closed. There is already an other issue created at site-core repository about the lookup fields. 

Thx for your time.
